app/form-components/team-form/TeamMembersForm.tsx

```diff
@@ -241,7 +247,7 @@ export const validateTeam = (team: { name: string; members: User[] }) => {
     else if (!validator.isEmail(member.idirEmail)) errors.members[i] = 'Please enter a valid email';
     else {
       let result = team.members.filter((email) => email.idirEmail === member.idirEmail).length;
-      if (result > 1) errors.members[i] = 'Please use unique email';
+      if (result > 1 || member.idirEmail === adminUser) errors.members[i] = 'Please use unique email';
```

Comment:
Could this line just check that `result > 1 || member.idirEmail === currentUser?.email` to avoid the global state? Or is that missing something

Reply:
It says, " reference Errror: currentUser is not defined" when you do that. That's why I have to create the new variable to capture the content to compare against. Not very nice, but I did not know how else.

Reply:
Oh I see, I thought that function was defined in the component, I see now it is a separate export. I think we might need to pass in the current user as a variable to the function then instead. The reason is just if someone tried to use that function without the component above having rendered first the current user would never get set into that variable which might lead to a real confusing state

Reply:
Yeah, that makes sense.

Reply:
Ok, I have done what you suggested, had to change 3 files. Because the validate function is called from 2 different locations. And I had to update the definition.

Reply:
thanks!